tests/queries/0_stateless/02049_clickhouse_local_merge_tree.expect

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/expect -f
 
+# Temporarily export expect logs into /test_output -- should get along with other artefacts into ci
+exp_internal -f /test_output/clickhouse_local_merge_tree_debug.log 0;
```

Comment:
BTW maybe all expect tests should be converted to include `exp_internal`?

Right now failures of tests with expect is hard to debug, since there is almost zero information, but this should help.

Reply:
This could be good, but for now I doubt this because files in /test_output  for functional tests are on the top of details link together for all tests. So having a file for each expect test might be too many, and I noticed exp_internal uses append for non-empty file, but this is for sure not an option.

Reply:
Right, not into /test_output (or at least make one archive from all of them)

Reply:
- https://github.com/ClickHouse/ClickHouse/pull/38728